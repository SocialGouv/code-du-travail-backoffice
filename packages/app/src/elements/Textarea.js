import styled from "@emotion/styled";

const Textarea = styled.textarea`
  -webkit-appearance: none;
  background-color: white;
  border: solid 1px var(--color-border);
  color: inherit;
  flex-grow: 1;
  font-family: inherit;
  font-weight: inherit;
  font-size: inherit;
  line-height: 1;
  padding: 0.4rem 0.6rem 0.575rem;
  resize: none;
`;

export default Textarea;
