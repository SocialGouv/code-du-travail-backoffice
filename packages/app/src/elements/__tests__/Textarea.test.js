import React from "react";
import { create } from "react-test-renderer";

import Textarea from "../Textarea";

describe("elements/<Textarea />", () => {
  const PROPS = {
    children: "A text",
  };

  it(`should pass`, () => {
    const props = {
      ...PROPS,
    };

    const $title = create(<Textarea {...props} />);

    expect($title).toHaveTestRenderedTextContent(props.children);
  });
});
