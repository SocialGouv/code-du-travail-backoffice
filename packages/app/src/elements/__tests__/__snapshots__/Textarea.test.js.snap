// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/<Textarea /> should match snapshot 1`] = `
.emotion-0 {
  -webkit-appearance: none;
  background-color: white;
  border: solid 1px var(--color-border);
  color: inherit;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  font-family: inherit;
  font-weight: inherit;
  font-size: inherit;
  line-height: 1;
  padding: 0.4rem 0.6rem 0.575rem;
  resize: none;
}

<div>
  <textarea
    class="emotion-0 emotion-1"
  />
</div>
`;
